doc/python/shapes.md

```diff
@@ -638,6 +638,34 @@ fig.update_layout(
 fig.show()
 ```
 
+#### Adding Shapes to Subplots
+Here we use the different axes (`x1`, `x2`) created by `make_subplots` as reference in order to draw shapes in figure subplots.
+```python
```

Comment:
maybe you need a blank line before the python cell here, not sure (please check)

Reply:
would try, not sure why CI fails